src/worktree.c

```diff
@@ -455,6 +455,26 @@ int git_worktree_is_locked(git_buf *reason, const git_worktree *wt)
 	return ret;
 }
 
+int git_worktree_name(const git_worktree *wt, git_buf *name)
+{
+	if (!name || !wt)
+		return -1;
+	
+	git_buf_clear(name);
+	return git_buf_puts(name, wt->name);	
+}
+
+int git_worktree_path(const git_worktree *wt, git_buf *path)
+{
+	if (!path || !wt)
+		return -1;
+	
+	git_buf_clear(path);
+	if ( git_path_dirname_r(path, wt->gitlink_path) > 0 )
```

Comment:
Please stick to existing coding style.

Reply:
@csware I would definitely like to. Could you tell me what specifically doesn't fit the coding style? Is it just the spaces after/before the parens in the if or something else too?